Update frontmatter for devdot (#31752)
hashicorp/terraform: website/docs/language/index.mdx

---
page_title: Overview - Configuration Language
description: >-

>   Use the Terraform language to write configuration files that tell
>   Terraform how to manage a collection of infrastructure.

Author: ```suggestion
  Use the Terraform configuration language to describe the infrastructure that Terraform manages.
```
